manifests/plugin/pulp.pp

```diff
@@ -64,10 +64,10 @@
     enabled       => $pulpnode_enabled,
     listen_on     => $listen_on,
   }
-  -> foreman_proxy::settings_file { 'pulp3':
-    template_path => 'foreman_proxy/plugin/pulp3.yml.erb',
+  -> foreman_proxy::settings_file { 'pulpcore':
```

Comment:
the gem currently has the config as `pulp3.yml`: https://github.com/theforeman/smart_proxy_pulp/blob/master/settings.d/pulp3.yml.example
and so does the RPM: https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec#L99

that means that with this change we will deploy an *additional* config, instead of updating the one from the existing installation.

and the plugin is still called `pulp3` in the code too: https://github.com/theforeman/smart_proxy_pulp/blob/master/lib/smart_proxy_pulp_plugin/pulp3_plugin.rb#L5, so renaming that file will break the plugin enablement.

I think the cheapest solution (for now) would to keep the settings file as `pulp3` until the plugin itself is renamed properly.

@jlsherrill might know of any plans around that.

Reply:
this was something i had wanted to do, can work on that today if we want to hold off a few hours.

Reply:
We don't need to rush, but would be cool if we can fix both sides ;)

Reply:
opened pr here: https://github.com/theforeman/smart_proxy_pulp/pull/22

Reply:
Marking as resolved since https://github.com/theforeman/smart_proxy_pulp/pull/22 and https://github.com/theforeman/foreman-packaging/pull/4553 are merged. Thanks!